fmriprep/utils/bids.py

```diff
@@ -193,3 +194,52 @@ def _grp_echos(x):
     subj_data.update({"bold": ses_uids})
 
     return subj_data, layout
+
+
+def write_derivative_description(bids_dir, deriv_dir):
+    from fmriprep import __version__
+
+    desc = {
+        'Name': 'fMRIPrep output',
+        'BIDSVersion': '1.1.1',
+        'PipelineDescription': {
+            'Name': 'fMRIPrep',
+            'Version': __version__,
+            # We could parse the version to reference a commit or tag
+            'CodeURL': 'https://github.com/poldracklab/fmriprep',
+            },
+        'CodeURL': 'https://github.com/poldracklab/fmriprep',
+        'HowToAcknowledge': 'https://github.com/poldracklab/fmriprep',
```

Comment:
Perhaps something more like current BIDS (sec 8.1.1)?:

```JSON
"HowToAcknowledge": "Please cite our paper (https://doi.org/10.1101/306951), and include the generated citation boilerplate within the Methods section of the text",
"ReferencesAndLinks": [ list generated from the boilerplate ],
"PipelineDOI": "https://doi.org/10.5281/zenodo.852659"
```

Reply:
PipelineDOI is not defined in the BIDS spec or the derivatives draft spec. ReferencesAndLinks is described as "List of references to publication that contain information on the dataset, or links.", which is different from how you seem to want to use it here. I updated `HowToAcknowledge`, as that seems fine. I'm a little inclined to try to move these things into `PipelineDescription` in the draft spec, to more clearly distinguish descriptions of the data and the tools that have been applied to it.

Reply:
You are right, I meant both `ReferencesAndLinks` and `PipelineDOI` to live under `PipelineDescription`. If moved there, then `PipelineDOI` might better be represented by `SoftwareDOI` (or similar, the idea being people setting the zenodo DOI here).